packages/storybook-utils/src/theme.ts

```diff
@@ -1,33 +1,40 @@
-import { ONYX_BREAKPOINTS as RAW_ONYX_BREAKPOINTS, type OnyxBreakpoint } from "sit-onyx";
-import onyxVariables from "sit-onyx/themes/onyx.json";
-import { create, type ThemeVars, type ThemeVarsPartial } from "storybook/internal/theming";
-import onyxLogo from "./assets/logo-onyx.svg";
+import {
+  ONYX_BREAKPOINTS as RAW_ONYX_BREAKPOINTS,
+  type OnyxBreakpoint,
+} from "@sit-onyx/shared/breakpoints";
+import { create, type ThemeVars } from "storybook/internal/theming";
+
+export type BrandDetails = Required<Pick<ThemeVars, "brandTitle" | "brandImage" | "brandUrl">>;
+
+/**
+ * Get the computed value for a CSS custom property.
+ * Per default the property value is taken from the body element.
+ */
+export const getCustomProperty = (property: string, el: Element = document.body) =>
+  getComputedStyle(el).getPropertyValue(property);
 
 /**
  * Creates a custom theme for Storybook that uses onyx colors.
  *
  * @see https://storybook.js.org/docs/react/configure/theming#create-a-theme-quickstart
  */
-export const createTheme = (
-  options?: Pick<ThemeVarsPartial, "base" | "brandTitle" | "brandImage" | "brandUrl">,
-) => {
-  const base = options?.base ?? "light";
-  const primaryColor = onyxVariables["onyx-color-themed-primary-500"];
+export const createTheme = (brandDetails: BrandDetails, base: "light" | "dark" = "light") => {
```

Comment:
Why is base not included in the brandDetails?

Reply:
Because the base is not related to the brand.